src/plone/api/tests/test_content.py

```diff
@@ -376,6 +376,64 @@ def test_create_at_with_title_in_request(self):
 
         self.assertEqual(page.title, 'Test document')
 
+    @unittest.skipIf(HAS_PACONTENTYPES, 'Archetypes only')
```

Comment:
@gbastien thinking minor, I think this test should run with dx too. Collection creation should work with dx. You can remove this condition and change the name of the test.